lib/views/home_content/feed/feed_sort_option_chips.dart

```diff
@@ -0,0 +1,42 @@
+import "package:flutter/material.dart";
+import "package:hooks_riverpod/hooks_riverpod.dart";
+import "package:proxima/services/sorting/post_sort_option.dart";
+import "package:proxima/viewmodels/feed_sort_options_view_model.dart";
+
+/// This widget is the list of sort options for the timeline.
+class FeedSortOptionChips extends ConsumerWidget {
+  static final optionChipKey = Map.fromIterables(
```

Comment:
Very small suggestion, I think we can make this variable to the plurial given that it is a map with multiple keys.
```suggestion
  static final optionChipKeys = Map.fromIterables(
```

Reply:
Fixed in f5420ecd18367f611c0e3939ef52571ab692c501.